devtools-ci-index.yaml

```diff
@@ -1667,6 +1667,24 @@
 
     <<: *eclipse-che-automation-template
 
+- job-template:
+    name: '{ci_project}-{git_repo}-che-integration-tests-rc-temporary'
+    wrappers:
+    - vault-secrets:
+        <<: *vault_defaults
+        secrets: 
+            - *quay-eclipse-che-credentials
```

Comment:
Do we need quay credentials here?
I thought this job would consume images already present on quay/dockerhub.

Reply:
Done.